infrastructure/storage_stack.py

```diff
@@ -102,4 +110,14 @@ def __init__(self, scope: Construct, stack_id: str, *, deploy_env: str, **kwargs
             ),
         )
 
+        ############################################################################################
+        # ### LAMBDA LAYERS BOTOCORE ###############################################################
+        ############################################################################################
+        self.botocore = aws_lambda_python.PythonLayerVersion(
```

Comment:
Does this actually belong in the storage stack?

Reply:
I'll try if moving botocore back to it's own stack is going to work

Reply:
`lambda_layers_stack` removed from `storage` and returned back to its own stack.